application/org.openjdk.jmc.updatesite.ide/src/main/resources/index.html

```diff
@@ -6,7 +6,7 @@
 -->
 <html>
   <head>
-    <title>JDK Mission Control for Eclipse</title>
+    <title>Eclipse: OpenJDK Plug-ins for JDK Mission Control</title>
```

Comment:
The plug-ins is really the Eclipse plug-ins for running JDK Mission Control in Eclipse + optional extra plug-ins. So, the most important quality of the plug-ins, is that it's JDK Mission Control for Eclipse. The second most important one is that it contains optional plug-ins for the plug-in version of JDK Mission Control. If you want to capture both, I guess saying "JDK Mission Control plug-ins for Eclipse" would kinda capture both.

I understand that you want to separate the Oracle plug-ins from the OpenJDK ones, but since OpenJDK is assumed as the default, I don't think that has to be stated.

Reply:
Do you suggest any other title Or should I keep it as it is?

Yes you are right the intent is to differentiate Oracle specific update sites.

Reply:
Then I think "JDK Mission Control Plug-ins for Eclipse". These are the default plug-ins, and don't need to be qualified. The Oracle ones would differ by stating that they are the Oracle ones. :)